queue/consumer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/ivcap-works/ivcap-service-sdk-python@queue_svc#egg=ivcap_sdk_service
```

Comment:
This is currently pinned to the `queue_svc` branch of the `ivcap-service-sdk-python`, which is ready for review. Once that branch gets merged into develop/main I will update these requirements files.

Reply:
Updated to point to `develop` now instead of `queue_svc`